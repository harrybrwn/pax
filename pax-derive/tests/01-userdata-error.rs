use pax_derive::UserData;

#[derive(UserData)]
enum Letters {
    A,
    B,
    C(i32),
}

fn main() {}
