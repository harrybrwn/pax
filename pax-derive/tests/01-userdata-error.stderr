error: expected pattern, found `,`
 --> tests/01-userdata-error.rs:3:10
  |
3 | #[derive(UserData)]
  |          ^^^^^^^^ expected pattern
  |
  = note: this error originates in the derive macro `UserData` (in Nightly builds, run with -Z macro-backtrace for more info)

error: proc-macro derive produced unparsable tokens
 --> tests/01-userdata-error.rs:3:10
  |
3 | #[derive(UserData)]
  |          ^^^^^^^^

error: only unit enums are supported, C is not a unit variant
 --> tests/01-userdata-error.rs:7:5
  |
7 |     C(i32),
  |     ^
